dcaf_case__management/Gemfile

```diff
@@ -0,0 +1,21 @@
+source 'https://rubygems.org'
+
+gem 'rails', '4.2.1'
+gem 'sqlite3'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.1.0'
+gem 'jquery-rails'
+gem 'turbolinks'
```

Comment:
I'm not a huge fan of turbolinks (possibly because I'm bad at javascript and it breaks js when you're bad at it). I'll make an issue on whether to remove and we can discuss.